commons/utils.js

```diff
@@ -176,12 +177,20 @@ const gasPriceWithinLimits = (gasPrice, limits) => {
 const normalizeGasPrice = (oracleGasPrice, factor, limits = null) => {
   let gasPrice = oracleGasPrice * factor
   gasPrice = gasPriceWithinLimits(gasPrice, limits)
-  return toBN(toWei(gasPrice.toFixed(2).toString(), 'gwei'))
+  return toWei(gasPrice.toFixed(2).toString(), 'gwei')
 }
 
-const gasPriceFromSupplier = async (url, options = {}) => {
+const gasPriceFromSupplier = async (web3, url, options = {}) => {
   try {
     let json
+    if (url === 'eip1559-gas-estimation') {
```

Comment:
Does it make sense to reflect this way to configure the gas price supplier [here](https://github.com/poanetwork/tokenbridge/blob/master/CONFIGURATION.md)?